src/content/docs/ru/tutorial/4-layouts/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+type: tutorial
+unitTitle: Сэкономьте время и энергию с помощью многоразовых макетов страниц
+title: 'Шаг 4 - Макеты (Layouts)'
+description: >-
+  Учебное пособие: Постройте свой первый блог с помощью Astro —
+
+  Используйте макеты Astro для общих элементов и стилей на всех своих страницах и
+  сообщениях блога
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Checklist from '~/components/Checklist.astro';
+import StackblitzIntro from '~/components/tutorial/StackblitzIntro.astro';
+
+
+Теперь, когда вы можете создавать компоненты, давайте создадим некоторые пользовательские макеты!
+
+## Где мы сейчас?
+
+<StackblitzIntro tree="withastro/blog-tutorial-demo/tree/unit-4/start"/>
```

Comment:
Like in the other chapters, you can write the equivalent Markdown and not use the component here!